src/javascript/app/base/page.js

```diff
@@ -54,7 +54,7 @@ const Page = (() => {
                 switch (evt.key) {
                     case 'active_loginid':
                         // not the active tab and logged out or loginid switch
-                        if (document.hidden && (evt.newValue === '' || !window.is_logging_in)) {
+                        if (evt.newValue === '' || !window.is_logging_in) {
```

Comment:
I guess the comment now has to be updated as well? why was the code there? does removing it break any other behaviour?

Reply:
That is a good question. I thought about that, why we added that in a first place.. will follow this up with you on slack.